rasterio/windows.py

```diff
@@ -648,7 +648,7 @@ def round_lengths(self, op='floor', pixel_precision=3):
         ----------
         op: str
             'ceil' or 'floor'
-        pixel_precision: int
+        pixel_precision: int, option (default: None)
```

Comment:
option -> optional

Reply:
@mwtoews thank you! Great to see you in here 😄